File: src/main/java/org/stt/persistence/stt/STTFile.java
package org.stt.persistence.stt;


import javax.inject.Qualifier;
import java.lang.annotation.Retention;
import java.lang.annotation.Target;

import static java.lang.annotation.ElementType.*;
import static java.lang.annotation.RetentionPolicy.RUNTIME;

/**
 * Created by dante on 04.12.14.
 */
@Qualifier
@Target({FIELD, PARAMETER, METHOD})
@Retention(RUNTIME)
public @interface STTFile {
}

File: src/main/kotlin/org/stt/persistence/stt/STTFile.kt
package org.stt.persistence.stt


import javax.inject.Qualifier

/**
 * Created by dante on 04.12.14.
 */
@Qualifier
@Target(AnnotationTarget.FIELD, AnnotationTarget.VALUE_PARAMETER, AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER)
@Retention()
annotation class STTFile
